scripts/checkpoint_downloader.py

```diff
@@ -0,0 +1,88 @@
+# Copyright 2018 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+"""
+Download all necessary files from a Magenta.js checkpoint URL.
+
+Example usage:
+
+$ python3 checkpoint_downloader /checkpoint/url/ /path/to/output
```

Comment:
Any reason for the python3 vs python mismatch between this and the script?

Reply:
oversight